Directory.Build.props

```diff
@@ -17,7 +17,7 @@
     <RepositoryUrl>https://github.com/dotnet/Orleans</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <IncludeSymbols>true</IncludeSymbols>
-    <IncludeSource>true</IncludeSource>
+    <IncludeSource>false</IncludeSource>
```

Comment:
This would affect the *.symbols.nupkg, right? I thought the plan was to include the sourcelink metadata, especially for the Microsoft Symbols server, but I don't think we should modify the symbols nuget packages. Whether we decide to not publish them later is another question, but those are still useful for people forking the repo I believe.

Reply:
Yes, that is right, ok, let me revert this.